### ISSUE TITLE
[bitnami/mongodb] No metrics besides `mongodb_up_1` exported

### ISSUE THREAD

**Issue by Anticom** [bitnami/charts#11404]
### Name and Version

bitnami/mongodb-exporter:0.31.2-debian-10-r14

### What steps will reproduce the bug?

We're using `bitnami/mongodb`(Version 11.2.0) helm chart with standalone architecture to run our MongoDB instance.

Here's the full values of the mongodb release:
```yaml
affinity: {}
annotations: {}
arbiter:
  affinity: {}
  annotations: {}
  args: []
  command: []
  configuration: ""
  containerSecurityContext:
    enabled: true
    runAsUser: 1001
  customLivenessProbe: {}
  customReadinessProbe: {}
  enabled: true
  existingConfigmap: ""
  extraEnvVars: []
  extraEnvVarsCM: ""
  extraEnvVarsSecret: ""
  extraFlags: []
  extraVolumeMounts: []
  extraVolumes: []
  hostAliases: []
  initContainers: []
  labels: {}
  livenessProbe:
    enabled: true
    failureThreshold: 6
    initialDelaySeconds: 30
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 5
  nodeAffinityPreset:
    key: ""
    type: ""
    values: []
  nodeSelector: {}
  pdb:
    create: false
    maxUnavailable: ""
    minAvailable: 1
  podAffinityPreset: ""
  podAnnotations: {}
  podAntiAffinityPreset: soft
  podLabels: {}
  podSecurityContext:
    enabled: true
    fsGroup: 1001
    sysctls: []
  priorityClassName: ""
  readinessProbe:
    enabled: true
    failureThreshold: 6
    initialDelaySeconds: 5
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 5
  resources:
    limits: {}
    requests: {}
  runtimeClassName: ""
  service:
    nameOverride: ""
  sidecars: []
  tolerations: []
architecture: standalone
args: []
auth:
  database: cms
  databases: []
  enabled: true
  existingSecret: ""
  password: <REDACTED>
  passwords: []
  replicaSetKey: ""
  rootPassword: <REDACTED>
  rootUser: root
  username: cms
  usernames: []
clusterDomain: cluster.local
command: []
common:
  exampleValue: common-chart
  global:
    imagePullSecrets: []
    imageRegistry: ""
    namespaceOverride: ""
    storageClass: ""
commonAnnotations: {}
commonLabels: {}
configuration: ""
containerSecurityContext:
  enabled: true
  runAsNonRoot: true
  runAsUser: 1001
customLivenessProbe: {}
customReadinessProbe: {}
customStartupProbe: {}
diagnosticMode:
  args:
  - infinity
  command:
  - sleep
  enabled: false
directoryPerDB: false
disableJavascript: false
disableSystemLog: false
enableIPv6: false
enableJournal: true
enabled: true
existingConfigmap: ""
externalAccess:
  autoDiscovery:
    enabled: false
    image:
      pullPolicy: IfNotPresent
      pullSecrets: []
      registry: docker.io
      repository: bitnami/kubectl
      tag: 1.23.6-debian-10-r1
    resources:
      limits: {}
      requests: {}
  enabled: false
  hidden:
    enabled: false
    service:
      annotations: {}
      domain: ""
      loadBalancerIPs: []
      loadBalancerSourceRanges: []
      nodePorts: []
      port: 27017
      type: LoadBalancer
  service:
    annotations: {}
    domain: ""
    loadBalancerIPs: []
    loadBalancerSourceRanges: []
    nodePorts: []
    port: 27017
    type: LoadBalancer
extraDeploy: []
extraEnvVars: []
extraEnvVarsCM: ""
extraEnvVarsSecret: ""
extraFlags: []
extraVolumeMounts: []
extraVolumes: []
fullnameOverride: ""
global:
  imagePullSecrets: []
  imageRegistry: ""
  namespaceOverride: ""
  storageClass: ""
hidden:
  affinity: {}
  annotations: {}
  args: []
  command: []
  configuration: ""
  customLivenessProbe: {}
  customReadinessProbe: {}
  enabled: false
  existingConfigmap: ""
  extraEnvVars: []
  extraEnvVarsCM: ""
  extraEnvVarsSecret: ""
  extraFlags: []
  extraVolumeMounts: []
  extraVolumes: []
  initContainers: []
  labels: {}
  livenessProbe:
    enabled: true
    failureThreshold: 6
    initialDelaySeconds: 30
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 5
  nodeAffinityPreset:
    key: ""
    type: ""
    values: []
  nodeSelector: {}
  pdb:
    create: false
    maxUnavailable: ""
    minAvailable: 1
  persistence:
    accessModes:
    - ReadWriteOnce
    annotations: {}
    enabled: true
    medium: ""
    mountPath: /bitnami/mongodb
    size: 8Gi
    storageClass: ""
    subPath: ""
    volumeClaimTemplates:
      dataSource: {}
      selector: {}
  podAffinityPreset: ""
  podAnnotations: {}
  podAntiAffinityPreset: soft
  podLabels: {}
  podManagementPolicy: OrderedReady
  priorityClassName: ""
  readinessProbe:
    enabled: true
    failureThreshold: 6
    initialDelaySeconds: 5
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 5
  replicaCount: 1
  resources:
    limits: {}
    requests: {}
  runtimeClassName: ""
  sidecars: []
  strategyType: RollingUpdate
  tolerations: []
hostAliases: []
image:
  debug: false
  pullPolicy: IfNotPresent
  pullSecrets: []
  registry: docker.io
  repository: bitnami/mongodb
  tag: 4.4.13-debian-10-r52
initContainers: []
initdbScripts: {}
initdbScriptsConfigMap: ""
labels: {}
livenessProbe:
  enabled: true
  failureThreshold: 6
  initialDelaySeconds: 30
  periodSeconds: 10
  successThreshold: 1
  timeoutSeconds: 5
metrics:
  containerPort: 9216
  enabled: true
  extraFlags: ""
  extraUri: ""
  image:
    pullPolicy: IfNotPresent
    pullSecrets: []
    registry: docker.io
    repository: bitnami/mongodb-exporter
    tag: 0.31.2-debian-10-r14
  livenessProbe:
    enabled: true
    failureThreshold: 3
    initialDelaySeconds: 15
    periodSeconds: 5
    successThreshold: 1
    timeoutSeconds: 5
  password: ""
  prometheusRule:
    additionalLabels: {}
    enabled: true
    namespace: ""
    rules:
    - alert: MongodbDown
      annotations:
        description: MongoDB instance is down.
        summary: MongoDB Down (instance {{ "{{" }} $labels.instance {{ "}}" }})
      expr: mongodb_up == 0
      for: 0m
      labels:
        severity: critical
    - alert: MongodbReplicaMemberUnhealthy
      annotations:
        description: MongoDB replica member is not healthy.
        summary: Mongodb replica member unhealthy (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: mongodb_rs_members_health == 0
      for: 0m
      labels:
        severity: critical
    - alert: MongodbReplicationLag
      annotations:
        description: Mongodb replication lag is more than 10s.
        summary: MongoDB replication lag (instance {{ "{{" }} $labels.instance {{
          "}}" }})
      expr: mongodb_rs_members_optimeDate{member_state="PRIMARY"} - on (set) group_right
        mongodb_rs_members_optimeDate{member_state="SECONDARY"} > 10
      for: 0m
      labels:
        severity: critical
    - alert: MongodbReplicationHeadroom
      annotations:
        description: MongoDB replication headroom is <= 0.
        summary: MongoDB replication headroom (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: sum(avg(mongodb_oplog_stats_start - mongodb_oplog_stats_end)) - sum(avg(mongodb_rs_members_optimeDate{member_state="PRIMARY"}
        - on (set) group_right mongodb_rs_members_optimeDate{member_state="SECONDARY"}))
        <= 0
      for: 0m
      labels:
        severity: critical
    - alert: MongodbNumberCursorsOpen
      annotations:
        description: Too many cursors opened by MongoDB for clients (> 10k).
        summary: MongoDB number cursors open (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: mongodb_ss_metrics_cursor_open{csr_type="total"} > 10 * 1000
      for: 2m
      labels:
        severity: warning
    - alert: MongodbCursorsTimeouts
      annotations:
        description: Too many cursors are timing out.
        summary: MongoDB cursors timeouts (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: increase(mongodb_ss_metrics_cursor_timedOut[1m]) > 100
      for: 2m
      labels:
        severity: warning
    - alert: MongodbTooManyConnections
      annotations:
        description: Too many connections (> 80%).
        summary: MongoDB too many connections (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: avg by(instance) (rate(mongodb_ss_connections{conn_type="current"}[1m]))
        / avg by(instance) (sum (mongodb_ss_connections) by (instance)) * 100 >
        80
      for: 2m
      labels:
        severity: warning
    - alert: MongodbVirtualMemoryUsage
      annotations:
        description: High memory usage.
        summary: MongoDB virtual memory usage (instance {{ "{{" }} $labels.instance
          {{ "}}" }})
      expr: (sum(mongodb_ss_mem_virtual) BY (instance) / sum(mongodb_ss_mem_resident)
        BY (instance)) > 3
      for: 2m
      labels:
        severity: warning
  readinessProbe:
    enabled: true
    failureThreshold: 3
    initialDelaySeconds: 5
    periodSeconds: 5
    successThreshold: 1
    timeoutSeconds: 1
  resources:
    limits: {}
    requests: {}
  service:
    annotations:
      prometheus.io/path: /metrics
      prometheus.io/port: '{{ .Values.metrics.service.port }}'
      prometheus.io/scrape: "true"
    port: 9216
    type: ClusterIP
  serviceMonitor:
    additionalLabels: {}
    enabled: true
    interval: 30s
    metricRelabelings: []
    namespace: ""
    relabellings: []
    scrapeTimeout: ""
  username: ""
nameOverride: ""
nodeAffinityPreset:
  key: ""
  type: ""
  values: []
nodeSelector: {}
pdb:
  create: false
  maxUnavailable: ""
  minAvailable: 1
persistence:
  accessModes:
  - ReadWriteOnce
  annotations: {}
  enabled: true
  existingClaim: ""
  medium: ""
  mountPath: /bitnami/mongodb
  size: 8Gi
  storageClass: ""
  subPath: ""
  volumeClaimTemplates:
    dataSource: {}
    requests: {}
    selector: {}
podAffinityPreset: ""
podAnnotations: {}
podAntiAffinityPreset: soft
podLabels: {}
podManagementPolicy: OrderedReady
podSecurityContext:
  enabled: true
  fsGroup: 1001
  sysctls: []
podSecurityPolicy:
  allowPrivilegeEscalation: false
  create: false
  privileged: false
  spec: {}
priorityClassName: ""
rbac:
  create: false
  role:
    rules: []
readinessProbe:
  enabled: true
  failureThreshold: 6
  initialDelaySeconds: 5
  periodSeconds: 10
  successThreshold: 1
  timeoutSeconds: 5
replicaCount: 2
replicaSetConfigurationSettings:
  configuration: {}
  enabled: false
replicaSetHostnames: true
replicaSetName: rs0
resources:
  limits:
    cpu: 2
    memory: 1Gi
  requests:
    cpu: 100m
    memory: 500Mi
runtimeClassName: ""
schedulerName: ""
service:
  annotations: {}
  clusterIP: ""
  externalIPs: []
  loadBalancerIP: ""
  loadBalancerSourceRanges: []
  nameOverride: ""
  nodePort: ""
  port: 27017
  portName: mongodb
  type: ClusterIP
serviceAccount:
  annotations: {}
  create: true
  name: ""
sidecars: []
startupProbe:
  enabled: false
  failureThreshold: 30
  initialDelaySeconds: 5
  periodSeconds: 10
  successThreshold: 1
  timeoutSeconds: 5
strategyType: RollingUpdate
systemLogVerbosity: 0
tls:
  autoGenerated: true
  caCert: ""
  caKey: ""
  enabled: false
  existingSecret: ""
  extraDnsNames: []
  image:
    pullPolicy: IfNotPresent
    registry: docker.io
    repository: bitnami/nginx
    tag: 1.21.6-debian-10-r85
  mode: requireTLS
tolerations: []
topologySpreadConstraints: []
useStatefulSet: false
volumePermissions:
  enabled: false
  image:
    pullPolicy: IfNotPresent
    pullSecrets: []
    registry: docker.io
    repository: bitnami/bitnami-shell
    tag: 10-debian-10-r404
  resources:
    limits: {}
    requests: {}
  securityContext:
    runAsUser: 0
```

### What is the expected behavior?

I'd expect to have all mongodb metrics exported.

### What do you see instead?

Besides all the `go_*` and `process_*` metrics, the only MongoDB related metric I get is `mongodb_up_1`:
```
# HELP go_gc_cycles_automatic_gc_cycles_total Count of completed GC cycles generated by the Go runtime.
# TYPE go_gc_cycles_automatic_gc_cycles_total counter
go_gc_cycles_automatic_gc_cycles_total 3544
# HELP go_gc_cycles_forced_gc_cycles_total Count of completed GC cycles forced by the application.
# TYPE go_gc_cycles_forced_gc_cycles_total counter
go_gc_cycles_forced_gc_cycles_total 0
# HELP go_gc_cycles_total_gc_cycles_total Count of all completed GC cycles.
# TYPE go_gc_cycles_total_gc_cycles_total counter
go_gc_cycles_total_gc_cycles_total 3544
# HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.
# TYPE go_gc_duration_seconds summary
go_gc_duration_seconds{quantile="0"} 3.1648e-05
go_gc_duration_seconds{quantile="0.25"} 5.5336e-05
go_gc_duration_seconds{quantile="0.5"} 7.6832e-05
go_gc_duration_seconds{quantile="0.75"} 0.000127147
go_gc_duration_seconds{quantile="1"} 0.00874866
go_gc_duration_seconds_sum 0.602974213
go_gc_duration_seconds_count 3544
# HELP go_gc_heap_allocs_by_size_bytes_total Distribution of heap allocations by approximate size. Note that this does not include tiny objects as defined by /gc/heap/tiny/allocs:objects, only tiny blocks.
# TYPE go_gc_heap_allocs_by_size_bytes_total histogram
go_gc_heap_allocs_by_size_bytes_total_bucket{le="8.999999999999998"} 1.001063e+06
go_gc_heap_allocs_by_size_bytes_total_bucket{le="24.999999999999996"} 1.0163618e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="64.99999999999999"} 1.5316193e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="144.99999999999997"} 1.8656149e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="320.99999999999994"} 1.9707242e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="704.9999999999999"} 2.0265826e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="1536.9999999999998"} 2.0363941e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="3200.9999999999995"} 2.0471741e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="6528.999999999999"} 2.0534314e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="13568.999999999998"} 2.0543548e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="27264.999999999996"} 2.0570174e+07
go_gc_heap_allocs_by_size_bytes_total_bucket{le="+Inf"} 2.0590457e+07
go_gc_heap_allocs_by_size_bytes_total_sum 5.764941256e+09
go_gc_heap_allocs_by_size_bytes_total_count 2.0590457e+07
# HELP go_gc_heap_allocs_bytes_total Cumulative sum of memory allocated to the heap by the application.
# TYPE go_gc_heap_allocs_bytes_total counter
go_gc_heap_allocs_bytes_total 5.764941256e+09
# HELP go_gc_heap_allocs_objects_total Cumulative count of heap allocations triggered by the application. Note that this does not include tiny objects as defined by /gc/heap/tiny/allocs:objects, only tiny blocks.
# TYPE go_gc_heap_allocs_objects_total counter
go_gc_heap_allocs_objects_total 2.0590457e+07
# HELP go_gc_heap_frees_by_size_bytes_total Distribution of freed heap allocations by approximate size. Note that this does not include tiny objects as defined by /gc/heap/tiny/allocs:objects, only tiny blocks.
# TYPE go_gc_heap_frees_by_size_bytes_total histogram
go_gc_heap_frees_by_size_bytes_total_bucket{le="8.999999999999998"} 996914
go_gc_heap_frees_by_size_bytes_total_bucket{le="24.999999999999996"} 1.0150963e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="64.99999999999999"} 1.529844e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="144.99999999999997"} 1.8634521e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="320.99999999999994"} 1.9683804e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="704.9999999999999"} 2.0241198e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="1536.9999999999998"} 2.0339128e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="3200.9999999999995"} 2.0446816e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="6528.999999999999"} 2.0509329e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="13568.999999999998"} 2.0518535e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="27264.999999999996"} 2.0545152e+07
go_gc_heap_frees_by_size_bytes_total_bucket{le="+Inf"} 2.0565419e+07
go_gc_heap_frees_by_size_bytes_total_sum 5.759278336e+09
go_gc_heap_frees_by_size_bytes_total_count 2.0565419e+07
# HELP go_gc_heap_frees_bytes_total Cumulative sum of heap memory freed by the garbage collector.
# TYPE go_gc_heap_frees_bytes_total counter
go_gc_heap_frees_bytes_total 5.759278336e+09
# HELP go_gc_heap_frees_objects_total Cumulative count of heap allocations whose storage was freed by the garbage collector. Note that this does not include tiny objects as defined by /gc/heap/tiny/allocs:objects, only tiny blocks.
# TYPE go_gc_heap_frees_objects_total counter
go_gc_heap_frees_objects_total 2.0565419e+07
# HELP go_gc_heap_goal_bytes Heap size target for the end of the GC cycle.
# TYPE go_gc_heap_goal_bytes gauge
go_gc_heap_goal_bytes 6.170784e+06
# HELP go_gc_heap_objects_objects Number of objects, live or unswept, occupying heap memory.
# TYPE go_gc_heap_objects_objects gauge
go_gc_heap_objects_objects 25038
# HELP go_gc_heap_tiny_allocs_objects_total Count of small allocations that are packed together into blocks. These allocations are counted separately from other allocations because each individual allocation is not tracked by the runtime, only their block. Each block is already accounted for in allocs-by-size and frees-by-size.
# TYPE go_gc_heap_tiny_allocs_objects_total counter
go_gc_heap_tiny_allocs_objects_total 2.184651e+06
# HELP go_gc_pauses_seconds_total Distribution individual GC-related stop-the-world pause latencies.
# TYPE go_gc_pauses_seconds_total histogram
go_gc_pauses_seconds_total_bucket{le="-5e-324"} 0
go_gc_pauses_seconds_total_bucket{le="9.999999999999999e-10"} 0
go_gc_pauses_seconds_total_bucket{le="9.999999999999999e-09"} 0
go_gc_pauses_seconds_total_bucket{le="9.999999999999998e-08"} 0
go_gc_pauses_seconds_total_bucket{le="1.0239999999999999e-06"} 0
go_gc_pauses_seconds_total_bucket{le="1.0239999999999999e-05"} 1155
go_gc_pauses_seconds_total_bucket{le="0.00010239999999999998"} 6642
go_gc_pauses_seconds_total_bucket{le="0.0010485759999999998"} 6987
go_gc_pauses_seconds_total_bucket{le="0.010485759999999998"} 7088
go_gc_pauses_seconds_total_bucket{le="0.10485759999999998"} 7088
go_gc_pauses_seconds_total_bucket{le="+Inf"} 7088
go_gc_pauses_seconds_total_sum NaN
go_gc_pauses_seconds_total_count 7088
# HELP go_goroutines Number of goroutines that currently exist.
# TYPE go_goroutines gauge
go_goroutines 20
# HELP go_info Information about the Go environment.
# TYPE go_info gauge
go_info{version="go1.17.8"} 1
# HELP go_memory_classes_heap_free_bytes Memory that is completely free and eligible to be returned to the underlying system, but has not been. This metric is the runtime's estimate of free address space that is backed by physical memory.
# TYPE go_memory_classes_heap_free_bytes gauge
go_memory_classes_heap_free_bytes 237568
# HELP go_memory_classes_heap_objects_bytes Memory occupied by live objects and dead objects that have not yet been marked free by the garbage collector.
# TYPE go_memory_classes_heap_objects_bytes gauge
go_memory_classes_heap_objects_bytes 5.66292e+06
# HELP go_memory_classes_heap_released_bytes Memory that is completely free and has been returned to the underlying system. This metric is the runtime's estimate of free address space that is still mapped into the process, but is not backed by physical memory.
# TYPE go_memory_classes_heap_released_bytes gauge
go_memory_classes_heap_released_bytes 8.159232e+06
# HELP go_memory_classes_heap_stacks_bytes Memory allocated from the heap that is reserved for stack space, whether or not it is currently in-use.
# TYPE go_memory_classes_heap_stacks_bytes gauge
go_memory_classes_heap_stacks_bytes 1.769472e+06
# HELP go_memory_classes_heap_unused_bytes Memory that is reserved for heap objects but is not currently used to hold heap objects.
# TYPE go_memory_classes_heap_unused_bytes gauge
go_memory_classes_heap_unused_bytes 948024
# HELP go_memory_classes_metadata_mcache_free_bytes Memory that is reserved for runtime mcache structures, but not in-use.
# TYPE go_memory_classes_metadata_mcache_free_bytes gauge
go_memory_classes_metadata_mcache_free_bytes 13568
# HELP go_memory_classes_metadata_mcache_inuse_bytes Memory that is occupied by runtime mcache structures that are currently being used.
# TYPE go_memory_classes_metadata_mcache_inuse_bytes gauge
go_memory_classes_metadata_mcache_inuse_bytes 19200
# HELP go_memory_classes_metadata_mspan_free_bytes Memory that is reserved for runtime mspan structures, but not in-use.
# TYPE go_memory_classes_metadata_mspan_free_bytes gauge
go_memory_classes_metadata_mspan_free_bytes 44136
# HELP go_memory_classes_metadata_mspan_inuse_bytes Memory that is occupied by runtime mspan structures that are currently being used.
# TYPE go_memory_classes_metadata_mspan_inuse_bytes gauge
go_memory_classes_metadata_mspan_inuse_bytes 218008
# HELP go_memory_classes_metadata_other_bytes Memory that is reserved for or used to hold runtime metadata.
# TYPE go_memory_classes_metadata_other_bytes gauge
go_memory_classes_metadata_other_bytes 5.055088e+06
# HELP go_memory_classes_os_stacks_bytes Stack memory allocated by the underlying operating system.
# TYPE go_memory_classes_os_stacks_bytes gauge
go_memory_classes_os_stacks_bytes 0
# HELP go_memory_classes_other_bytes Memory used by execution trace buffers, structures for debugging the runtime, finalizer and profiler specials, and more.
# TYPE go_memory_classes_other_bytes gauge
go_memory_classes_other_bytes 3.274377e+06
# HELP go_memory_classes_profiling_buckets_bytes Memory that is used by the stack trace hash map used for profiling.
# TYPE go_memory_classes_profiling_buckets_bytes gauge
go_memory_classes_profiling_buckets_bytes 1.682191e+06
# HELP go_memory_classes_total_bytes All memory mapped by the Go runtime into the current process as read-write. Note that this does not include memory mapped by code called via cgo or via the syscall package. Sum of all metrics in /memory/classes.
# TYPE go_memory_classes_total_bytes gauge
go_memory_classes_total_bytes 2.7083784e+07
# HELP go_memstats_alloc_bytes Number of bytes allocated and still in use.
# TYPE go_memstats_alloc_bytes gauge
go_memstats_alloc_bytes 5.66292e+06
# HELP go_memstats_alloc_bytes_total Total number of bytes allocated, even if freed.
# TYPE go_memstats_alloc_bytes_total counter
go_memstats_alloc_bytes_total 5.764941256e+09
# HELP go_memstats_buck_hash_sys_bytes Number of bytes used by the profiling bucket hash table.
# TYPE go_memstats_buck_hash_sys_bytes gauge
go_memstats_buck_hash_sys_bytes 1.682191e+06
# HELP go_memstats_frees_total Total number of frees.
# TYPE go_memstats_frees_total counter
go_memstats_frees_total 2.275007e+07
# HELP go_memstats_gc_cpu_fraction The fraction of this program's available CPU time used by the GC since the program started.
# TYPE go_memstats_gc_cpu_fraction gauge
go_memstats_gc_cpu_fraction 0
# HELP go_memstats_gc_sys_bytes Number of bytes used for garbage collection system metadata.
# TYPE go_memstats_gc_sys_bytes gauge
go_memstats_gc_sys_bytes 5.055088e+06
# HELP go_memstats_heap_alloc_bytes Number of heap bytes allocated and still in use.
# TYPE go_memstats_heap_alloc_bytes gauge
go_memstats_heap_alloc_bytes 5.66292e+06
# HELP go_memstats_heap_idle_bytes Number of heap bytes waiting to be used.
# TYPE go_memstats_heap_idle_bytes gauge
go_memstats_heap_idle_bytes 8.3968e+06
# HELP go_memstats_heap_inuse_bytes Number of heap bytes that are in use.
# TYPE go_memstats_heap_inuse_bytes gauge
go_memstats_heap_inuse_bytes 6.610944e+06
# HELP go_memstats_heap_objects Number of allocated objects.
# TYPE go_memstats_heap_objects gauge
go_memstats_heap_objects 25038
# HELP go_memstats_heap_released_bytes Number of heap bytes released to OS.
# TYPE go_memstats_heap_released_bytes gauge
go_memstats_heap_released_bytes 8.159232e+06
# HELP go_memstats_heap_sys_bytes Number of heap bytes obtained from system.
# TYPE go_memstats_heap_sys_bytes gauge
go_memstats_heap_sys_bytes 1.5007744e+07
# HELP go_memstats_last_gc_time_seconds Number of seconds since 1970 of last garbage collection.
# TYPE go_memstats_last_gc_time_seconds gauge
go_memstats_last_gc_time_seconds 1.65787912574863e+09
# HELP go_memstats_lookups_total Total number of pointer lookups.
# TYPE go_memstats_lookups_total counter
go_memstats_lookups_total 0
# HELP go_memstats_mallocs_total Total number of mallocs.
# TYPE go_memstats_mallocs_total counter
go_memstats_mallocs_total 2.2775108e+07
# HELP go_memstats_mcache_inuse_bytes Number of bytes in use by mcache structures.
# TYPE go_memstats_mcache_inuse_bytes gauge
go_memstats_mcache_inuse_bytes 19200
# HELP go_memstats_mcache_sys_bytes Number of bytes used for mcache structures obtained from system.
# TYPE go_memstats_mcache_sys_bytes gauge
go_memstats_mcache_sys_bytes 32768
# HELP go_memstats_mspan_inuse_bytes Number of bytes in use by mspan structures.
# TYPE go_memstats_mspan_inuse_bytes gauge
go_memstats_mspan_inuse_bytes 218008
# HELP go_memstats_mspan_sys_bytes Number of bytes used for mspan structures obtained from system.
# TYPE go_memstats_mspan_sys_bytes gauge
go_memstats_mspan_sys_bytes 262144
# HELP go_memstats_next_gc_bytes Number of heap bytes when next garbage collection will take place.
# TYPE go_memstats_next_gc_bytes gauge
go_memstats_next_gc_bytes 6.170784e+06
# HELP go_memstats_other_sys_bytes Number of bytes used for other system allocations.
# TYPE go_memstats_other_sys_bytes gauge
go_memstats_other_sys_bytes 3.274377e+06
# HELP go_memstats_stack_inuse_bytes Number of bytes in use by the stack allocator.
# TYPE go_memstats_stack_inuse_bytes gauge
go_memstats_stack_inuse_bytes 1.769472e+06
# HELP go_memstats_stack_sys_bytes Number of bytes obtained from system for stack allocator.
# TYPE go_memstats_stack_sys_bytes gauge
go_memstats_stack_sys_bytes 1.769472e+06
# HELP go_memstats_sys_bytes Number of bytes obtained from system.
# TYPE go_memstats_sys_bytes gauge
go_memstats_sys_bytes 2.7083784e+07
# HELP go_sched_goroutines_goroutines Count of live goroutines.
# TYPE go_sched_goroutines_goroutines gauge
go_sched_goroutines_goroutines 20
# HELP go_sched_latencies_seconds Distribution of the time goroutines have spent in the scheduler in a runnable state before actually running.
# TYPE go_sched_latencies_seconds histogram
go_sched_latencies_seconds_bucket{le="-5e-324"} 0
go_sched_latencies_seconds_bucket{le="9.999999999999999e-10"} 204800
go_sched_latencies_seconds_bucket{le="9.999999999999999e-09"} 204800
go_sched_latencies_seconds_bucket{le="9.999999999999998e-08"} 212308
go_sched_latencies_seconds_bucket{le="1.0239999999999999e-06"} 335615
go_sched_latencies_seconds_bucket{le="1.0239999999999999e-05"} 415327
go_sched_latencies_seconds_bucket{le="0.00010239999999999998"} 531501
go_sched_latencies_seconds_bucket{le="0.0010485759999999998"} 537348
go_sched_latencies_seconds_bucket{le="0.010485759999999998"} 537671
go_sched_latencies_seconds_bucket{le="0.10485759999999998"} 537677
go_sched_latencies_seconds_bucket{le="+Inf"} 537677
go_sched_latencies_seconds_sum NaN
go_sched_latencies_seconds_count 537677
# HELP go_threads Number of OS threads created.
# TYPE go_threads gauge
go_threads 18
# HELP mongodb_up Whether MongoDB is up.
# TYPE mongodb_up gauge
mongodb_up 1
# HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
# TYPE process_cpu_seconds_total counter
process_cpu_seconds_total 3790.58
# HELP process_max_fds Maximum number of open file descriptors.
# TYPE process_max_fds gauge
process_max_fds 1.048576e+06
# HELP process_open_fds Number of open file descriptors.
# TYPE process_open_fds gauge
process_open_fds 17
# HELP process_resident_memory_bytes Resident memory size in bytes.
# TYPE process_resident_memory_bytes gauge
process_resident_memory_bytes 2.418688e+07
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.65771941138e+09
# HELP process_virtual_memory_bytes Virtual memory size in bytes.
# TYPE process_virtual_memory_bytes gauge
process_virtual_memory_bytes 7.3996288e+08
# HELP process_virtual_memory_max_bytes Maximum amount of virtual memory available in bytes.
# TYPE process_virtual_memory_max_bytes gauge
process_virtual_memory_max_bytes 1.8446744073709552e+19
```

### Additional information

First of all I'm not sure whether this is the correct repository to report the bug to. If not please let me know which one to forward it to.

---

Furthermore I can't see any issues with the k8s deployment itself. Here's an excerpt from `kubectl describe` of the mongodb pod:
```
  metrics:
    Container ID:  docker://b0cb3a62c15373fc0355693226003e425f7c9cb38e3bf83fa3ae96882e6c600f
    Image:         docker.io/bitnami/mongodb-exporter:0.31.2-debian-10-r14
    Image ID:      docker-pullable://bitnami/mongodb-exporter@sha256:815713f441bc80fc1f27e7e385eaac27c1363dee6cc6847f11e7e540a0786e9a
    Port:          9216/TCP
    Host Port:     0/TCP
    Command:
      /bin/bash
      -ec
    Args:
      /bin/mongodb_exporter --web.listen-address ":9216" --mongodb.uri "mongodb://$MONGODB_ROOT_USER:$(echo $MONGODB_ROOT_PASSWORD | sed -r "s/@/%40/g;s/:/%3A/g")@localhost:27017/admin?"

    State:          Running
      Started:      Wed, 13 Jul 2022 15:36:52 +0200
    Ready:          True
    Restart Count:  0
    Liveness:       http-get http://:metrics/ delay=15s timeout=5s period=5s #success=1 #failure=3
    Readiness:      http-get http://:metrics/ delay=5s timeout=1s period=5s #success=1 #failure=3
    Environment:
      MONGODB_ROOT_USER:      root
      MONGODB_ROOT_PASSWORD:  <set to the key 'mongodb-root-password' in secret 'raccoon-mongodb'>  Optional: false
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-s9xr2 (ro)
```

And here's the entire log from the metrics sidecar:
```
2022/07/13 13:36:52 Starting HTTP server for http://:9216/metrics ...
```

**Comment by Anticom** [comment]
The issue is still relevant to us

**Comment by chary1112004** [comment]
@Anticom have you tried to upgrade helm chart at least to 12.1.24?

**Comment by Anticom** [comment]
@chary1112004 No I haven't. Does this version fix the issue?

**Comment by chary1112004** [comment]
@Anticom yes. From this version fixes the issue.

**Comment by Anticom** [comment]
@chary1112004 Cheers, I'll give it a shot next week and keep you posted!

**Comment by fmulero** [comment]
Some issues were detected after the upgrade to mongodb-exporter 0.33, some of them were fixed in version 12.1.26. Please try version higher than that one
